src/library/scala/collection/convert/DecorateAsJava.scala

```diff
@@ -18,91 +18,91 @@ import scala.language.implicitConversions
 trait DecorateAsJava extends AsJavaConverters {
   /**
    * Adds an `asJava` method that implicitly converts a Scala `Iterator` to a Java `Iterator`.
-   * See [[asJavaIterator]].
+   * @see [[asJavaIterator]]
    */
   implicit def asJavaIteratorConverter[A](i : Iterator[A]): AsJava[ju.Iterator[A]] =
     new AsJava(asJavaIterator(i))
 
   /**
-   * Adds an `asJavaEnumeration` method that implicitly converts a Scala `Iterator` to a Java
-   * `Enumeration`. See [[asJavaEnumeration]].
+   * Adds an `asJavaEnumeration` method that implicitly converts a Scala `Iterator` to a Java `Enumeration`.
+   * @see [[asJavaEnumeration]]
```

Comment:
Tested in browser and works to the extend Scaladoc links currently work. A separate issue remains shown in this video,

> https://youtu.be/-ZVMGxwAAkM

Clicking the generated link hides the target method instead of continuing to show it. Not a fix for this PR however.